modules/currency-l0/src/main/scala/org/tessellation/cli/snapshot.scala

```diff
@@ -0,0 +1,17 @@
+package org.tessellation.cli
+
+import org.tessellation.config.types.SnapshotConfig
+import org.tessellation.ext.decline.decline._
+
+import com.monovore.decline._
+import eu.timepit.refined.auto._
+import fs2.io.file.Path
+
+object snapshot {
+
+  val snapshotPath: Opts[Path] = Opts
+    .env[Path]("CL_SNAPSHOT_STORED_PATH", help = "Path to store created snapshot")
+    .withDefault(Path("data/snapshot")) // TODO: Should path be prefixed to prevent overriding of another SC?
```

Comment:
imo each jar file should be executed in separate working directory. I think it should be responsibility of the runner to set working directory or correct paths 🤔

Reply:
I totally agree with that